src/db/ClimbTypes.ts

```diff
@@ -103,6 +103,7 @@ export enum SafetyType {
 export interface IGradeType {
   yds?: string
   ewbank?: string
+  brazilian_crux?: string
```

Comment:
just a nitpick - can you please use camelCase?  Our linter (ts-standard) is supposed to catch it but it doesn't for some reason.

https://standardjs.com/rules.html